roles/download/templates/kubeadm-images.yaml.j2

```diff
@@ -1,5 +1,13 @@
 {% if kube_version is version('v1.12.0', '>=') %}
```

Comment:
Due to this condition the v1alpha1 and v1alpha2 blocks can be removed from here

Reply:
Thanks, you are right.